src/frontend/src/lib/components/networks/NetworkButton.svelte

```diff
@@ -26,8 +27,8 @@
 	};
 </script>
 
-<button class="w-full flex justify-between items-center" on:click={onClick}>
-	<TextWithLogo {name} {icon} logo="start" />
+<button class="w-full flex justify-between items-start" on:click={onClick}>
```

Comment:
`items-start` is maybe the default? Would removing this class as the same result?

Reply:
Yes, forgot to remove this one. It's done now.